sonaranalyzer-dotnet/rspec/cs/Sonar_way_profile.json

```diff
@@ -1,174 +1,175 @@
-    "S3450",
-    "S3451",
-    "S3453",
-    "S3456",
-    "S3457",
-    "S3458",
-    "S3459",
-    "S3466",
-    "S3597",
-    "S3598",
-    "S3600",
-    "S3603",
-    "S3604",
-    "S3610",
-    "S3626",
-    "S3655",
-    "S3776",
-    "S3869",
-    "S3871",
-    "S3875",
-    "S3877",
-    "S3881",
-    "S3884",
-    "S3885",
-    "S3887",
-    "S3889",
-    "S3897",
-    "S3903",
-    "S3904",
-    "S3925",
-    "S3926",
-    "S3927",
-    "S3928",
-    "S3971",
-    "S3981",
-    "S3984",
-    "S3998",
-    "S4015",
-    "S4016",
-    "S4019",
-    "S4035"
-  ]
+{
+  "name": "Sonar way",
+  "ruleKeys": [
+    "S101",
+    "S107",
+    "S108",
+    "S110",
+    "S112",
+    "S125",
+    "S818",
+    "S907",
+    "S927",
+    "S1006",
+    "S1066",
+    "S1104",
+    "S1116",
+    "S1117",
+    "S1118",
+    "S1121",
+    "S1123",
+    "S1125",
+    "S1134",
+    "S1135",
+    "S1144",
+    "S1155",
+    "S1163",
+    "S1172",
+    "S1185",
+    "S1186",
+    "S1206",
+    "S1210",
+    "S1215",
+    "S1450",
+    "S1479",
+    "S1481",
+    "S1643",
+    "S1656",
+    "S1699",
+    "S1751",
+    "S1764",
+    "S1848",
+    "S1854",
+    "S1862",
+    "S1871",
+    "S1905",
+    "S1939",
+    "S1940",
+    "S1944",
+    "S2068",
+    "S2123",
+    "S2178",
+    "S2184",
+    "S2190",
+    "S2201",
+    "S2219",
+    "S2223",
+    "S2225",
+    "S2234",
+    "S2259",
+    "S2275",
+    "S2278",
+    "S2290",
+    "S2291",
+    "S2292",
+    "S2306",
+    "S2326",
+    "S2328",
+    "S2342",
+    "S2344",
+    "S2345",
+    "S2346",
+    "S2365",
+    "S2368",
+    "S2372",
+    "S2376",
+    "S2386",
+    "S2436",
+    "S2437",
+    "S2486",
+    "S2583",
+    "S2589",
+    "S2681",
+    "S2688",
+    "S2692",
+    "S2696",
+    "S2737",
+    "S2743",
+    "S2757",
+    "S2758",
+    "S2761",
+    "S2930",
+    "S2933",
+    "S2934",
+    "S2953",
+    "S2971",
+    "S2995",
+    "S2996",
+    "S2997",
+    "S3005",
+    "S3010",
+    "S3168",
+    "S3169",
+    "S3172",
+    "S3217",
+    "S3218",
+    "S3220",
+    "S3236",
+    "S3237",
+    "S3241",
+    "S3244",
+    "S3246",
+    "S3247",
+    "S3249",
+    "S3251",
+    "S3256",
+    "S3261",
+    "S3262",
+    "S3263",
+    "S3264",
+    "S3265",
+    "S3346",
+    "S3376",
+    "S3397",
+    "S3427",
+    "S3440",
+    "S3442",
+    "S3443",
+    "S3444",
+    "S3445",
+    "S3447",
+    "S3449",
+    "S3450",
+    "S3451",
+    "S3453",
+    "S3456",
+    "S3457",
+    "S3458",
+    "S3459",
+    "S3466",
+    "S3597",
+    "S3598",
+    "S3600",
+    "S3603",
+    "S3604",
+    "S3610",
+    "S3626",
+    "S3655",
+    "S3776",
+    "S3869",
+    "S3871",
+    "S3875",
+    "S3877",
+    "S3881",
+    "S3884",
+    "S3885",
+    "S3887",
+    "S3889",
+    "S3897",
+    "S3903",
+    "S3904",
+    "S3925",
+    "S3926",
+    "S3927",
+    "S3928",
+    "S3971",
+    "S3972",
+    "S3981",
+    "S3984",
+    "S3998",
+    "S4015",
+    "S4016",
+    "S4019",
+    "S4035"
+  ]
```

Comment:
Wasn't this file excluded from SCM? Otherwise I think we need to sync our line ending settings...